ui/component/channelEdit/view.jsx

```diff
@@ -22,6 +22,7 @@ import SUPPORTED_LANGUAGES from 'constants/supported_languages';
 import WalletSpendableBalanceHelp from 'component/walletSpendableBalanceHelp';
 import { SIMPLE_SITE } from 'config';
 import { sortLanguageMap } from 'util/default-languages';
+import ThumbnailBrokenImage from './thumbnail-broken.png';
```

Comment:
Don't think this needs a broken png uploaded? https://github.com/lbryio/lbry-desktop/search?q=thumbnail-broken